test/test_read/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+file(GLOB TEST_SRC_FILES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+add_executable(run_test_read ${TEST_SRC_FILES})
+target_link_libraries(run_test_read gtest_main ReadLib)
+
+add_custom_command(TARGET run_test_read POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy_directory
+        "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build/bin/Debug/"
+        "${CMAKE_BINARY_DIR}/test/test_read/Debug/")
```

Comment:
linux (and maybe macos) doesn't have Debug folder, so it should be covered in
```cmake
if (WIN32)
    ...
endif()
```
else we will clone from `${CMAKE_SOURCE_DIR}/3rdparty/opencv/build/bin` to `${CMAKE_BINARY_DIR}/test/test_read/`, i think